paths/release_triggers/create.yaml

```diff
@@ -0,0 +1,91 @@
+---
+summary: Create a release trigger
+description: Create a new recurring release. New releases will be published automatically, based on the cron schedule provided.
+  Currently, only one release trigger can exist per distribution.
+operationId: release_triggers/create
+tags:
+  - Release Triggers
+parameters:
+  - "$ref": "../../parameters.yaml#/X-PhraseApp-OTP"
+  - "$ref": "../../parameters.yaml#/account_id"
+  - "$ref": "../../parameters.yaml#/distribution_id"
+requestBody:
+  required: true
+  content:
+    application/json:
+      schema:
+        type: object
+        title: release/create/parameters
+        properties:
+          cron_schedule:
+            description: Cron schedule for the scheduler. Read more about the format of this field at https://en.wikipedia.org/wiki/Cron
+            type: string
+            example: "15 18 1,3 * *"
+          time_zone:
+            description: Time zone for the scheduler
+            type: string
+            example: "Europe/Berlin"
+          locale_ids:
+            description: List of locale ids that will be included in the release. If empty, distribution locales will be used
```

Comment:
Actually thats not the case atm, we are not falling back to distribution's locales if the params are nil. Should we? 🤔

Reply:
I'll remove that detail for now, I believe it should fall back, but again, a product decision I guess.